shell/platform/windows/flutter_windows_view_unittests.cc

```diff
@@ -1287,6 +1287,13 @@ TEST(FlutterWindowsViewTest, UpdatesVSyncOnDwmUpdates) {
   std::unique_ptr<MockAngleSurfaceManager> surface_manager =
       std::make_unique<MockAngleSurfaceManager>();
 
```

Comment:
Is there a straightforward way to verify the release on the platform thread after surface creation, so that no one reverts that bit?

Reply:
Added!